README.sp.md

```diff
@@ -0,0 +1,746 @@
+
+# scrcpy (v1.17)
+
+Esta aplicación proporciona imagen y control de un dispositivo Android conectado
+por USB (o [por TCP/IP][article-tcpip]). No requiere acceso _root_.
+Compatible con _GNU/Linux_, _Windows_ y _macOS_.
+
+![screenshot](assets/screenshot-debian-600.jpg)
+
+Sus características principales son:
+
+ - **ligero** (nativo, solo muestra la imagen del dispositivo)
+ - **desempeño** (30~60fps)
+ - **calidad** (1920×1080 o superior)
+ - **baja latencia** ([35~70ms][lowlatency])
+ - **corto tiempo de inicio** (~1 segundo para mostrar la primera imagen)
+ - **no intrusivo** (no se deja nada instalado en el dispositivo)
+
+[lowlatency]: https://github.com/Genymobile/scrcpy/pull/646
+
+
+## Requisitos
+
+El dispositivo Android requiere como mínimo API 21 (Android 5.0).
+
+Asegurate de [habilitar el adb debugging][enable-adb] en tu(s) dispositivo(s).
+
+[enable-adb]: https://developer.android.com/studio/command-line/adb.html#Enabling
+
+En algunos dispositivos, también necesitas habilitar [una opción adicional][control] para controlarlo con el teclado y ratón.
+
+[control]: https://github.com/Genymobile/scrcpy/issues/70#issuecomment-373286323
+
+
+## Consigue la app
+
+<a href="https://repology.org/project/scrcpy/versions"><img src="https://repology.org/badge/vertical-allrepos/scrcpy.svg" alt="Packaging status" align="right"></a>
+
+### Resumen
+
+ - Linux: `apt install scrcpy`
+ - Windows: [download][direct-win64]
+ - macOS: `brew install scrcpy`
+
+Construir desde la fuente: [BUILD] ([proceso simplificado][BUILD_simple])
+
+[BUILD]: BUILD.md
+[BUILD_simple]: BUILD.md#simple
+
+
+### Linux
+
+En Debian (_test_ y _sid_ por ahora) y Ubuntu (20.04):
+
+```
+apt install scrcpy
+```
+
+Hay un paquete [Snap]: [`scrcpy`][snap-link].
+
+[snap-link]: https://snapstats.org/snaps/scrcpy
+
+[snap]: https://en.wikipedia.org/wiki/Snappy_(package_manager)
+
+Para Fedora, hay un paquete [COPR]: [`scrcpy`][copr-link].
+
+[COPR]: https://fedoraproject.org/wiki/Category:Copr
+[copr-link]: https://copr.fedorainfracloud.org/coprs/zeno/scrcpy/
+
+Para Arch Linux, hay un paquete [AUR]: [`scrcpy`][aur-link].
+
+[AUR]: https://wiki.archlinux.org/index.php/Arch_User_Repository
+[aur-link]: https://aur.archlinux.org/packages/scrcpy/
+
+Para Gentoo, hay un paquete [Ebuild]: [`scrcpy/`][ebuild-link].
+
+[Ebuild]: https://wiki.gentoo.org/wiki/Ebuild
+[ebuild-link]: https://github.com/maggu2810/maggu2810-overlay/tree/master/app-mobilephone/scrcpy
+
+También puedes [construir la aplicación manualmente][BUILD] ([proceso simplificado][BUILD_simple]).
+
+
+### Windows
+
+Para Windows, por simplicidad, hay un pre-compilado con todas las dependencias
+(incluyendo `adb`):
+
+ - [`scrcpy-win64-v1.17.zip`][direct-win64]  
+   _(SHA-256: 8b9e57993c707367ed10ebfe0e1ef563c7a29d9af4a355cd8b6a52a317c73eea)_
+
+[direct-win64]: https://github.com/Genymobile/scrcpy/releases/download/v1.17/scrcpy-win64-v1.17.zip
```

Comment:
```suggestion
 - [README](README.md#windows) 
```

Or something similar, the other translations have been modified like this in order to always have the last download release.

Reply:
Thanks, I merged your suggestion